### DISCUSSION TITLE
incorrect display of tags <details> and <summary>

### DISCUSSION THREAD

**Discussion by Aphr0di** [requarks/wiki#3589] [Error / Bug Report]
when I create a page containing these tags in markdown, the content of `<summary>` does not fit within the `<details>`

![1](https://user-images.githubusercontent.com/68544018/87932385-412f6700-caad-11ea-9c34-46d47f75ed59.PNG)

**Comment by n1xx1** [comment]
The problem here is that the styling is incorrect. The selectors are `... details summary { ... }` instead of `... details > summary { ... }`, so opening the parent cause a lot of issues in the childrens.

I opened this pull request with the fix: #4156
